software/Makefile

```diff
@@ -44,7 +46,7 @@ build_beebs: build_libs
 # run this after you build the beebs to create nbf files (x86 server)
 generate_and_packup_nbf: $(riscv-binaries:.riscv=.nbf)
 	mkdir -p nbf
-	-cp -f $^ nbf
+	-mv -f $^ nbf
```

Comment:
I think this causes it to regenerate the nbfs each time? Probably fine, but what’s the motivation

Reply:
The cp leaves nbf files in the SDK directories. The change to search for binaries in only sdk/prog might make this less of an issue, but I'm personally not a fan of the sdk pollution. It quickly becomes unclear whether a lingering nbf file in the sdk tree is up to date or not (or in sync with the files in software/nbf/

Reply:
Fine with me